src/update.js

```diff
@@ -40,7 +58,19 @@ const generateUpdateExpression = unless(
   // The `original` argument is assumed to be an empty object so only `SET`
   // expressions are supported by default
   // @see https://github.com/4ossiblellc/dynamodb-update-expression/blob/master/README.md#usage
-  compose(wrapOver('ExpressionAttributeValues'), partial(getUpdateExpression, [{}]))
+  compose(wrapOver('ExpressionAttributeValues'), params => {
+    return evolve(
+      {
+        ExpressionAttributeNames: compose(
+          // NOTE: DynamoDB doesn't support special charecters in the `UpdateExpression` string and the `dynamodb-update-expression` doesn't handle this cases,
+          // in order to support, at least, camel-cases object key is necessary to transform them into camelCase instead.
+          zipObjWithValues(Object.keys(params)),
+          keys
```

Comment:
```suggestion
          Object.keys
```

Or use `keys` in the call above to keep it consistent.